src/Form/FormAnalyzer.js

```diff
@@ -41,7 +41,7 @@ class FormAnalyzer {
          */
         this.signals = []
 
-        this.evaluateElAttributes(input, 3, true)
+        this.evaluateElAttributes(input, 1, true)
```

Comment:
This has a very minor impact, rightly so, and it's lowered further here. It's a legacy from the very first implementation when we were only expecting single-field forms and in fact it only runs on the very first field we encounter for each form. I played around with it, trying to actually run this on all input fields, removing it, etc. Ultimately, keeping it for only one field and lowering its impact yielded the best result.